operators-examples/qiskit-dev-operator/operator/manifests/qiskit-dev-operator.v0.1.0.clusterserviceversion.yaml

```diff
@@ -0,0 +1,175 @@
+apiVersion: operators.coreos.com/v1alpha1
```

Comment:
what is the difference between your manifests dir and your 0.1.0 dir - are they the same?

Reply:
I created manifests to push the files to quay. `0.1.0 ` was created by `operator-sdk generate csv`. Is it ok to delete `0.1.0`?